librz/core/cmd_quit.c

```diff
@@ -15,6 +15,34 @@ static const char *help_msg_q[] = {
 	NULL
 };
 
+RZ_IPI RzCmdStatus rz_cmd_quit_handler(RzCore *core, int argc, const char **argv) {
+	core->num->value = 0LL;
+	//exit (*input?rz_num_math (core->num, input+1):0);
+	//if (core->http_up) return false; // cancel quit when http is running
+	return RZ_CMD_STATUS_EXIT;
+}
+
+RZ_IPI RzCmdStatus rz_quit_kill_save_handler(RzCore *core, int argc, const char **argv) {
+	core->num->value = 5;
```

Comment:
Use some defines for these values (which I'm not sure what they mean... but let's keep them for now).
For example, RZ_QUIT_VALUE_KILL, RZ_QUIT_VALUE_SAVE, RZ_QUIT_VALUE_NOKILL, etc.